modules/CompanyUser/Services/CompanyUserCRUDService.php

```
@@ -58,16 +67,21 @@ public function create(CreateCompanyUserDTO $createCompanyUserDTO, CreateCompany
 
     public function sendEmailAssignToCompanyToUser($user , $companyId)
     {
-        $userInCompany = $this->userRepository->findOneBy(["global_company_user_id" => $user->global_id])->first();
-        $companyId = (string)$companyId;
-        $company = $this->companyRepository->getCompany(Uuid::fromString($companyId));
-        $data = [
-            "name" => $userInCompany->name,
-            "company_name" => $company->name,
-            "domain_name" => "https://".$company->domains()->first()?->domain,
-            "serial_no" => $company->serial_no
-        ];
-        $userInCompany->notify(new SendDomainForUser($data));
+        try {
+            $userInCompany = $this->userRepository->findOneBy(["global_company_user_id" => $user->global_id])->first();
+            $companyId = (string)$companyId;
+            $company = $this->companyRepository->getCompany(Uuid::fromString($companyId));
+            $data = [
+                "name" => $userInCompany->name,
+                "company_name" => $company->name,
+                "domain_name" => "https://".$company->domains()->first()?->domain,
+                "serial_no" => $company->serial_no
+            ];
+            $userInCompany->notify(new SendDomainForUser($data));
+        } catch (\Exception $e) {
+            // Re-throw the exception so callers can handle it if needed
+            throw $e;
```

Simply re-throwing the exception without any additional context or logging doesn't provide any value. Either:
1. Remove the try-catch block entirely to let exceptions bubble up naturally
2. Add proper error handling (logging, transformation to domain-specific exceptions, etc.)